### PR TITLE
Bypass links: allow keyboard-only users to skip navigation

### PR DESCRIPTION
**Pull request by lhenze**
### Description

* Adds new block to all pages for a bypass link
* Adds CSS to hide the block except when it's focused

closes https://github.com/inveniosoftware/invenio-app-rdm/issues/1171

### Checklist

Ticks in all boxes and 🟢 on all GitHub actions status checks are required to merge:

- [x] I'm aware of the [code of conduct](https://inveniordm.docs.cern.ch/contribute/code-of-conduct/).
- [x] I've created [logical separate commits](https://inveniordm.docs.cern.ch/develop/best-practices/commits/#commits) and followed the [commit message format](https://inveniordm.docs.cern.ch/develop/best-practices/commits/#commit-message).
- [ ] I've added relevant test cases.
- [ ] I've added relevant documentation.
- [x] I've marked [translation strings](https://inveniordm.docs.cern.ch/develop/best-practices/i18n/) (for relevant code).
- [x] I've followed the [CSS/JS](https://inveniordm.docs.cern.ch/develop/best-practices/css-js/) and [React](https://inveniordm.docs.cern.ch/develop/best-practices/react/) guidelines (for relevant code).
- [x] I've followed the [web accessibility](https://inveniordm.docs.cern.ch/develop/best-practices/accessibility/) guidelines (for relevant code).
- [x] I've followed the [user interface](https://inveniordm.docs.cern.ch/develop/best-practices/ui/) guidelines (for relevant code).
- [x] I've identified the [copyright holder(s)](https://inveniordm.docs.cern.ch/contribute/copyright-policy/) and updated copyright headers for touched files (>15 lines contributions).
- [x] I've NOT included third-party code (copy/pasted source code or new dependencies).

### CODE REVIEW

**Comment by lhenze** [comment]
Hi, 
Here is some more information: 
https://webaim.org/techniques/skipnav/
https://www.w3.org/TR/WCAG20/#navigation-mechanisms-skip

And a screenshot of expected behavior
Screen Shot 2022-03-18 at 9 19 11 AM

File: invenio_theme/assets/semantic-ui/less/invenio_theme/theme/globals/site.variables [hunk 1]
```diff
@@ -34,3 +34,8 @@
 
 @headerFont        : @fontName, sans-serif;
 @pageFont          : @fontName, sans-serif;
+
+/*--- Focus ---*/
+
+@focusColor : #3dbbdb;
+@focusBackgroundColor : inherit;
```
Comment (kpsherva) [inline invenio_theme/assets/semantic-ui/less/invenio_theme/theme/globals/site.variables:41]: I don't see the usage of this variable, am I missing something? If it is not used, it should be removed

File: invenio_theme/assets/semantic-ui/less/invenio_theme/theme/globals/site.overrides [hunk 2]
```diff
@@ -190,3 +190,35 @@ code {
 }
 
 .generate-truncate-classes(5);
+
+*:focus, input:focus, select:focus, button:focus {
+  outline: 3px solid @focusColor !important;
+}
```
[the remark this answers is not in the document]
Comment (lhenze) [reply invenio_theme/assets/semantic-ui/less/invenio_theme/theme/globals/site.overrides:196]: Ok, sounds good, in fact I'll take this part out completely; it doesn't really belong with this issue.

**Review by jennur — APPROVED** [review]

**Comment by lhenze** [comment]
screenshot with skip to main button
Improved design now uses existing styles and is left-aligned with a small top and left margin.

**Review by kpsherva — APPROVED** [review]

**Pull request merged**